docker-entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+nohup poetry run flask run --host 0.0.0.0 &
```

Comment:
interesting approach. the drawback is that it's problematic to distinguish worker and service logs, they both go into one stdout. better to have worker as yet another container in your docker compose. it should use the same Dockerfile. overload the command in docker-compose:
 ```
 worker:
    build:
      context: .
      dockerfile: Dockerfile
    command: poetry run python worker.py # or use worker.sh script
    depends_on:
      - db
      - redis
    environment:
      REDIS_HOST: redis
```